UnityProject/Assets/Scripts/Core/Sprite Handler/RandomInitialSprite.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Core.Sprites
+{
+	/// <summary>
+	/// Initializes the object with a random sprite SO from the SO list defined in SpriteHandler component.
+	/// </summary>
+	public class RandomInitialSprite : MonoBehaviour
```

Comment:
you could probably include this as part of Sprite handler may be?

Reply:
Done; please confirm changes to `SpriteHandler` look good.